src/odemis/driver/test/smatpod_test.py

```diff
@@ -42,8 +42,8 @@
         "role": "",
         "ref_on_init": True,
         "actuator_speed": 0.1,  # m/s
-        "locator": "usb:ix:0",
-        # "locator": "fake",
+        # "locator": "usb:ix:0",
+        "locator": "fake",
```

Comment:
Please use "usb:ix:0", and if `TEST_NOHW` is set then override "locator" to "fake".